shared/teams/add-members-wizard/add-from-where.tsx

```diff
@@ -22,6 +22,8 @@ const AddFromWhere = (props: Props) => {
 
   const onClose = () => dispatch(nav.safeNavigateUpPayload())
   const onContinueKeybase = () => dispatch(appendNewTeamBuilder(teamID))
+  const onContinuePhone = () =>
```

Comment:
I feel like this isn't really useful to have without also putting the data in the store. But I guess that's happening in a future ticket anyway so /shrug

Reply:
I have some of the store setup started in https://github.com/keybase/client/pull/22957.